src/Backupper/AbstractBackupper.php

```diff
@@ -16,13 +17,18 @@ abstract class AbstractBackupper implements \IteratorAggregate
 {
     protected ?string $destination = null;
     protected array $excludedTables = [];
+    protected string $defaultOptions = '';
```

Comment:
As you initialize the property in constructor, I don't know if this is not better:
```suggestion
    protected string $defaultOptions;
```

Reply:
(maybe we don't care)

Reply:
I always ask myself this same question every time I meet this case. I don't know what is the best practice.